src/components/ManagerForm/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import Button from "../Button";
+import FormGroup from "../FormGroup";
+import FormHeading from "../FormHeading";
+import Label from "../Label";
+import Form from "../Form";
+import SelectStatus from "../SelectStatus";
+
+const ManagerForm = ({ errors, manager = {}, submit }) => {
+  const [status, setStatus] = useState(manager?.status);
+
+  const action = manager.uuid ? "Edit" : "Add";
+
+  const hasError = (field) =>
+    errors.find((error) => error.id === `${field}-error`);
+
+  const errorMessage = (field) => {
+    const error = errors.filter((err) => err.id === `${field}-error`);
+    return error.length === 1 ? error[0].message : "";
+  };
```

Comment:
These methods have also been extracted into common helpers in `src/helpers/pageErrorHandler`, could you use them instead please?

Reply:
I just removed validation error checking in current ManagerForm as it only consists of a select component which has fixed dropdown values that does not need validating. Will not be using the error helper functions for this particular component.